File: 344. Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        tail_ind = len(s) - 1
        head_ind = 0
        
        while(tail_ind > head_ind):
            temp = s[head_ind]
            s[head_ind] = s[tail_ind]
            s[tail_ind] = temp

            head_ind += 1
            tail_ind -= 1

File: 344. Reverse String.c
void reverseString(char *s, int sSize)
{
    int tail_ind = sSize - 1;
    int head_ind = 0;

    while (tail_ind > head_ind)
    {
        char temp = s[head_ind];
        s[head_ind] = s[tail_ind];
        s[tail_ind] = temp;

        head_ind++;
        tail_ind--;
    }
}